res/skins/Deere/style.qss

```diff
@@ -278,40 +353,21 @@ QTreeView::branch:open:has-children:has-siblings {
 }
 
 /* QSplitter between LibrarySidebar and Library */
-#LibrarySplitter::handle {
+QSplitter::handle {
```

Comment:
LibrarySplitter (vertical) and  LibraryCoverArtSplitter (horizontal) used to use different images, now they use both the verticall image.